src/Refactoring-Core/RBExtractToTemporaryRefactoring.class.st

```diff
@@ -43,12 +43,12 @@ RBExtractToTemporaryRefactoring class >> model: aRBSmalltalk extract: anInterval
 RBExtractToTemporaryRefactoring >> applicabilityPreconditions [
 
 	^ {
-			(ReDefinesSelectorsCondition new definesSelectors: (Array with: selector) in: class).
-		  	(ReIsValidInstanceVariableName new name: newVariableName).
+			(ReDefinesSelectorsCondition new definesSelectors: { selector } in: class).
+			(ReIsValidInstanceVariableName new name: newVariableName).
 			(ReCheckVariableNameCondition
-				class: class
-				variableName: newVariableName 
-				parseTree: parseTree ).
+				class: class 
+      			variableName: newVariableName 
+     			parseTree: parseTree)
```

Comment:
Now that you have `preconditionValidInstVarName` and `preconditionCheckVarName` you can use them here as well, right?